src/components/Comment/ReplyingToIndicator/ReplyingToIndicator.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { IndicatorWrapper, AbortButton } from './styled';
+type ReplyingToIndicatorProps = {
+  replyingToAuthor: string | undefined;
+  setReplyToCommentId: Function;
+  setReplyToCommentAuthor: Function;
+};
+
+function ReplyingToIndicator({
+  replyingToAuthor,
+  setReplyToCommentId,
+  setReplyToCommentAuthor,
+}: ReplyingToIndicatorProps) {
+  function handleClick() {
+    setReplyToCommentId(undefined);
+    setReplyToCommentAuthor(undefined);
```

Comment:
This should probably also be replaced with `onAbort`. This component doesn't need to know about comment ID. This is important as you may then change the behaviour later without having to make changes in all of these sub-components